src/app/modules/filters/filters-popover/filters-popover.component.html

```diff
@@ -7,8 +7,8 @@
       <mat-icon svgIcon="filter" class="icon"></mat-icon>
     </ng-template>
   </button>
-  <div class="popup-box slide-in" *ngIf="filtersBoxVisible">
-    <div class="popup-body fade-in" *ngIf="filtersVisible">
+  <div id="popupContainer">
```

Comment:
Prefer classes over id
```suggestion
  <div class="popupContainer" #filterContainer>
```

(#filterContainer is for use with @ViewChild)